### PR TITLE
Inline node support for Slate + Link implementation

### PR DESCRIPTION
**Pull request by skogsmaskin**

### CODE REVIEW

File: src/inputs/BlockEditor-slate/toolbar/LinkButton.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  }
+
+  handleChange = event => {
+    const value = event.target.value
+    this.setState({href: value})
+  }
+
+
+  render() {
+    const onClose = () => {
```
Comment (bjoerge) [inline src/inputs/BlockEditor-slate/toolbar/LinkButton.js]: Should not create functions inside render. Can you add this as an instance method instead?

File: src/inputs/BlockEditor-slate/toolbar/LinkButton.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        Object.assign(
+          {popupOpen: false},
+          this.linkDataFromProps()
+        )
+      )
+    }
+    return (
+      <div>
+        <ToggleButton
+          onClick={() => this.setState({popupOpen: true})}
```
Comment (bjoerge) [inline src/inputs/BlockEditor-slate/toolbar/LinkButton.js]: Same here

File: src/inputs/BlockEditor-slate/toolbar/LinkButton.js (hunk 1 above)
Comment (skogsmaskin) [inline src/inputs/BlockEditor-slate/toolbar/LinkButton.js]: Done

File: src/inputs/BlockEditor-slate/toolbar/LinkButton.js (hunk 2 above)
Comment (skogsmaskin) [inline src/inputs/BlockEditor-slate/toolbar/LinkButton.js]: Done

**Comment by skogsmaskin** [comment]
Slate inline nodes and links

**Pull request merged**